### ISSUE TITLE
Feature Request: Fixed Scale

### ISSUE THREAD

**Issue by JamesBelchamber** [ccoors/Valeronoi#33]
Just used this to map out my house (awesome) and the different maps have different scales (from highest to lowest detected dBa) but, of course, the same colours. This means that comparing signal between e.g. the upstairs and downstairs map is not so easy.

Would it be possible to add a setting that allows manual configuration of the weakest and strongest signal? If it's an acceptable feature I might have a crack at adding it, because technically I'm guessing it's not going to be that hard (..famous last words, eh).

**Comment by ccoors** [comment]
Good idea, that would definitely be a useful feature. Sure, you can give it a try, I don't know when I'll have time to look into it.
